src/modules/launcher/PowerLauncher/MainWindow.xaml.cs

```diff
@@ -475,21 +491,90 @@ private void ClearAutoCompleteText(TextBox textBox)
             {
                 SearchBox.AutoCompleteTextBlock.Text = string.Empty;
             }
+
+            var showResultsWithDelay = _viewModel.GetSearchQueryResultsWithDelaySetting();
+
+            if (showResultsWithDelay)
+            {
+                // Default means we don't do anything we did not do before... leave the results as is, they will be changed as needed when results are returned
+                var pTRunStartNewSearchAction = _settings.PTRunStartNewSearchAction ?? "Default";
+
+                if (pTRunStartNewSearchAction == "DeSelect")
```

Comment:
I like "DeSelect" option. Looks more natural than others.

Reply:
I agree "DeSelect" is the better if _settings.PTRSearchQueryFastResultsWithDelay = true. 


But, what I think is the **best configuration** is this, because we get fast results and zero flicker. 
- _settings.PTRunStartNewSearchAction == "Default" (the way it has always been done)
-  _settings.PTRSearchQueryFastResultsWithDelay == false (do fast stuff without throttle, slow stuff with throttle)

The second best options is, it does prevent the issue, but is not as good as above.
- _settings.PTRunStartNewSearchAction == "DeSelect" (deselect all results while new results are inbound)
-  _settings.PTRSearchQueryFastResultsWithDelay == true (throttle everything)
 
I made one other change in this area. Let me know if not resoved